aws-ssm-patchbaseline/template.yml

```diff
@@ -4,19 +4,34 @@ Description: AWS SAM template for the AWS::SSM::PatchBaseline resource type
 
 Globals:
   Function:
-    Timeout: 60  # docker start-up times can be long for SAM CLI
+    Timeout: 30  # docker start-up times can be long for SAM CLI
+    MemorySize: 256
 
 Resources:
   TypeFunction:
     Type: AWS::Serverless::Function
     Properties:
       Handler: software.amazon.ssm.patchbaseline.HandlerWrapper::handleRequest
-      Runtime: java8
       CodeUri: ./target/aws-ssm-patchbaseline-handler-1.0-SNAPSHOT.jar
+      Runtime: java8
+      MemorySize: 512
+      Environment:
+       Variables:
+          PARAM1: VALUE
 
-  TestEntrypoint:
+  HelloWorldFunction:
     Type: AWS::Serverless::Function
     Properties:
-      Handler: software.amazon.ssm.patchbaseline.HandlerWrapper::testEntrypoint
-      Runtime: java8
       CodeUri: ./target/aws-ssm-patchbaseline-handler-1.0-SNAPSHOT.jar
+      Handler: software.amazon.ssm.patchbaseline.App::handleRequest
```

Comment:
i see it's being used here, but why do you think this is needed? from CFN team requirement? Idon't see it's being implemented in other services, is this a  new requirement. (for HandlerWrapper and App file)

I actually don't know if this template.yml actually has any functionality

please explain more

Reply:
yes the above changes are used for CFN test only and i have validated the-changes with template.yaml file.

The template.yaml will used to trigger the CFN test entry point. In the template file we are pointing the handlerwrapper which will observe the handler event dynamically based contract test generation process.
Once the CFN test invoke, it will trigger the handler events to initiate the appropriate handler option and will do the test cases.

After the trigger the process if process gets all the need data it will be sucess orelse it will get false.

Reply:
I don't think we need this, this is complicating the package, i think we can ignore this in the package, but you can confirm with CFN team

Reply:
i will coordinate with CFN team then will remove this line as well.

Reply:
do we have any resolution on this?